plugins/woocommerce/src/Internal/Admin/Orders/Edit.php

```diff
@@ -492,14 +497,17 @@ class="metabox-holder columns-<?php echo ( 1 === get_current_screen()->get_colum
 	 * Helper function to render meta boxes.
 	 */
 	private function render_meta_boxes() {
+		$order_post = get_post( $this->order->get_id() );
 		?>
 		<div id="postbox-container-1" class="postbox-container">
 			<?php do_meta_boxes( $this->screen_id, 'side', $this->order ); ?>
 		</div>
 		<div id="postbox-container-2" class="postbox-container">
 			<?php
-			do_meta_boxes( $this->screen_id, 'normal', $this->order );
-			do_meta_boxes( $this->screen_id, 'advanced', $this->order );
+			if ( $order_post instanceof WP_Post ) {
+				do_meta_boxes( $this->screen_id, 'normal', $order_post );
+				do_meta_boxes( $this->screen_id, 'advanced', $order_post );
```

Comment:
The risk seems different here, and the third parameter (in a call made to `do_meta_boxes`) is the `$data_object`, which is not necessarily a `WP_Post`. Does this section make as much sense?

Reply:
Yes, you are definitely correct, In My tests I have used `WP_Post` as the metabox callback parameter, but that's wrong, I reverted the change in here.